addons/mail/static/src/models/thread/thread.js

```diff
@@ -664,6 +665,7 @@ registerModel({
                     followers: insertAndReplace(followersData.map(followerData =>
                         this.messaging.models['Follower'].convertData(followerData)
                     )),
+                    hasWriteAccess,
```

Comment:
Doesn't need to be inside `followersData` condition, just add it into `values` at the start.